README/ReleaseNotes/v636/index.md

```diff
@@ -59,6 +59,7 @@ The following people have contributed to this new version:
 - When running multiple computation graphs run concurrently using [`RunGraphs()`](https://root.cern/doc/master/namespaceROOT_1_1RDF.html#a526d77d018bf69462d736bbdd1a695c4),
   the pool of slot numbers that a thread can pick from is now shared across all graphs. This enables use cases where a single resource, which may be expensive to create or copy,
   is shared across tasks belonging to different computation graphs.
+- Support for single-threaded snapshotting to RNTuple has been added. This can be enabled through `RSnapshotOptions`. Note that snapshotting from a TTree-based RDataFrame to RNTuple is not yet supported in this release, but will be added in a future version. The current recommended way to convert from TTree to RNTuple is through the [RNTupleImporter](https://root.cern/doc/v636/classROOT_1_1Experimental_1_1RNTupleImporter.html).
```

Comment:
I'm probably again warming up a discussion we had multiple times, but what was needed to from a TTree-based RDataFrame to RNTuple? My last memory is that it may just work because RDF turns collections into `RVec`s...

Reply:
I looked into it very briefly but I need to refresh my memory on whether there were any blockers or not. I will take a look again but the change will likely be too large to still backport for this release..